scammer-label-propagation/README.md

```diff
@@ -0,0 +1,33 @@
+# Large Tether Transfer Agent
+
+## Description
+
+This agent propagates scammer labels towards unlabeled addresses. The flow is as follows:
+1. A new label is published in an alert by one of the bots this agent is subscribed to. If the confidence is over a certain threshold, it triggers the whole process
+1. Based on the address that was labeled as attacker, it collects the first order neighbours (any address that had a transaction from/to the central node). From every neighbour, it collects a representation of the address with aggregated measures, and of the transactions between any two addresses in the graph.
+1. Using forta alerts, and subscribed bots, all the labels (both positive and negative) for any of the addresses are collected, and will be used afterwards.
+1. Using the previous data a Graph Neural Network is trained. As the model don't have all the information from every node, the agent uses semi-supervised learning to learn from the known labels, and then it predicts on all the addresses that there is no information. From within those, when the model is confident (over a parameter) than an address is an attacker, this address is the published as findings.
```

Comment:
i recommend to add a bit more technical description on the type of model to this readme